src/plugins.js

```diff
@@ -37,21 +39,22 @@ function load(plugin, options, file) {
  *
  * @param {Object} config PostCSS Config Plugins
  *
- * @return {Array} plugins PostCSS Plugins
+ * @return {Promise<Array>} plugins PostCSS Plugins
  */
-function plugins(config, file) {
+async function plugins(config, file) {
   let list = []
 
   if (Array.isArray(config.plugins)) {
     list = config.plugins.filter(Boolean)
   } else {
-    list = Object.keys(config.plugins)
-      .filter(plugin => {
-        return config.plugins[plugin] !== false ? plugin : ''
+    list = Object.entries(config.plugins)
+      .filter(([plugin, options]) => {
+        return options !== false ? plugin : false
```

Comment:
We should add a test for this. I had thought about removing it, then I realized why it's written this way.

Reply:
Sure. We can add in this PR.

Reply:
Thinking again, why is this written this way? 😅

The only edge case here is if option is true but the plugin is empty string.

I don't think that will ever happen. Simply this should also work

```ts
      .filter(([, options]) => {
        return options !== false
      }
```